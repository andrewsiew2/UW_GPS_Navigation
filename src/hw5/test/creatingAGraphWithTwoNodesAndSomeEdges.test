# Test if two nodes can be created in a graph with edges

# Create a graph
CreateGraph graphWithTwoNodes

# Add a pair of nodes
AddNode graphWithTwoNodes NodeA
AddNode graphWithTwoNodes NodeB

# Add an edge
AddEdge graphWithTwoNodes NodeA NodeB labelA
AddEdge graphWithTwoNodes NodeB NodeA labelB

# Print the nodes in the graph and the outgoing edges
ListNodes graphWithTwoNodes
ListChildren graphWithTwoNodes NodeA
ListChildren graphWithTwoNodes NodeB