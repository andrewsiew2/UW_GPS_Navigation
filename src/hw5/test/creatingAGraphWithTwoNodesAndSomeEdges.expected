# Test if two nodes can be created in a graph with edges

# Create a graph
created graph graphWithTwoNodes

# Add a pair of nodes
added node NodeA to graphWithTwoNodes
added node NodeB to graphWithTwoNodes

# Add an edge
added edge labelA from NodeA to NodeB in graphWithTwoNodes
added edge labelB from NodeB to NodeA in graphWithTwoNodes

# Print the nodes in the graph and the outgoing edges 
graphWithTwoNodes contains: NodeA NodeB
the children of NodeA in graphWithTwoNodes are: NodeB(labelA)
the children of NodeB in graphWithTwoNodes are: NodeA(labelB)